.gitignore

```diff
@@ -1 +1,3 @@
 spec/tmp*
+.idea/
+pkg/
```

Comment:
if you put this into your global .gitignore you will never have to add it to any project https://github.com/grosser/dotfiles/blob/master/gitconfig#L66-L67